enforcer/README.md

```diff
@@ -0,0 +1,218 @@
+# KRR Enforcer - Kubernetes Resource Recommendation Mutation Webhook
+
+A mutating webhook server that automatically enforces [KRR (Kubernetes Resource Recommender)](https://github.com/robusta-dev/krr) recommendations by patching pod resource requests and limits in real-time.
+
+## Features
+
+- **Automatic Resource Enforcement**: Applies KRR recommendations to pods during pod creation
+- **Flexible Enforcement Modes**: Support for enforce/ignore modes per workload
+- **REST API**: Query recommendations via HTTP endpoints
+
+## Enforcement Modes
+
+The enforcer supports three enforcement modes:
```

Comment:
This is a little confusing. I suggest

> Enforcement can be configured globally or on a per-workload basis.

And then rename "Default Mode" below to "Global Enforcement Mode"